src/convertToObject.js

```diff
@@ -12,7 +12,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesList = sourceString.split('\n');
+
+  const stylesObj = {};
+
+  stylesList.forEach((style, ind, arr) => {
+    arr[ind] = style.split(':');
+
+    if (arr[ind].length > 1) {
+      arr[ind][0] = arr[ind][0].trim();
+      // console.log(arr[ind][1]);
+      arr[ind][1] = arr[ind][1].slice(0, arr[ind][1].length - 1).trim();
+    } else {
+      arr[ind][0] = arr[ind][0].trim();
+    }
+    stylesObj[arr[ind][0]] = arr[ind][1];
+  });
```

Comment:
let's separate concerns here
first, prepare an array like this (you can use array iteration methods here instead of double indexes)
`[ [ background-color, #fff] , [-webkit-border-radius, 5px] ]`

and then in a separate loop create an object out of it